recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ source:
     - hardcoded-paths.patch
 
 build:
-  number: 2
+  number: 0
   skip: True  # [win]
```

Comment:
Sorry I overlooked this before. We should pick a Python to use for the build. As it is not a run time dependency, we don't need 3 builds with different Pythons. They should all be the same. Would you mind picking one and re-rendering again to drop the Python matrix.